queries/update_table_metadata.py

```
@@ -0,0 +1,88 @@
+                f"A descrição da da tabela '{table_id}' tem mais de 16384 caracteres, não é possível atualizar."
+            )
+        else:
+            print(f"Atualizando a descrição da tabela '{table_id}'")
+            tabela["description"] = descricao_tabela
+
+    for i in range(len(tabela["schema"]["fields"])):
+        field = tabela["schema"]["fields"][i]
+        if field["name"] in descricoes_colunas and descricoes_colunas[field["name"]]:
+            if (
+                "description" in field and field["description"] != descricoes_colunas[field["name"]]
+            ) or "description" not in field:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)